webapp/components/wpt-amend-metadata.js

```diff
@@ -252,13 +252,26 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
           if (!(test in link)) {
             link[test] = [];
           }
-          link[test].push({ 'url': entry.url, 'product': entry.product });
+          const metadata = { 'url': entry.url };
+          if (entry.product !== '') {
+            metadata['product'] = entry.product;
+          }
+          link[test].push(metadata);
         }
       }
     }
     return link;
   }
 
+  getProductLogo(product) {
```

Comment:
<img width="746" alt="Screen Shot 2021-02-23 at 12 37 23 PM" src="https://user-images.githubusercontent.com/8611520/108886467-ff344980-75d6-11eb-810a-979d71eb7f5f.png">

The icons aren't showing for the wpt-metadata entries on the bottom either. I think we need to change `this.displayLogo(product)` instead; [see wpt-metadata.js](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/components/wpt-metadata.js#L45)

Reply:
Remove this method and put the same code to `displayLogo`